### PR TITLE
Adding Gimlet

### PR DESCRIPTION
**Pull request by laszlocph**
If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
image
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?
* [x] Have you included the specific version of the application to be installed?

Matching documentation piece is available:

https://gimlet.io/docs/installing-gimlet-on-civo-cloud

### CODE REVIEW

**Review by kunal-kushwaha — APPROVED** [review]
lgtm

File: gimlet/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Gimlet
+version: installer-v0.4.2
```
Comment (saiyam1814) [inline gimlet/manifest.yaml:2]: Can we just have v0.4.2?

File: gimlet/install.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+helm repo add onechart https://chart.onechart.dev
+
+kubectl create namespace gimlet-installer || true
+
+helm repo update \
+ && helm upgrade gimlet-installer --install onechart/onechart \
+    --namespace gimlet-installer  \
+    -f values.yaml \
```
Comment (saiyam1814) [inline gimlet/install.sh:10]: -f values.yaml works from the folder ?

File: gimlet/manifest.yaml (hunk 1 above)
Comment (laszlocph) [reply gimlet/manifest.yaml]: Yes, updated.

File: gimlet/install.sh (hunk 2 above)
Comment (laszlocph) [reply gimlet/install.sh]: I removed the values.yaml and used set for all parameters. The values yaml may not be propagated by the CIVO automation.

**Comment by saiyam1814** [comment]
LGTM

**Pull request merged**